hibernate-reactive-core/src/main/java/org/hibernate/reactive/engine/ReactiveActionQueue.java

```diff
@@ -326,12 +325,13 @@ public void clear() {
 		}
 	}
 
-	public CompletionStage<Void> addAction(ReactiveEntityRegularInsertAction action) {
+	public CompletionStage<Void> addAction(ReactiveEntityInsertAction action) {
```

Comment:
Why did you change the signature of this method?

Reply:
mistake. fixed